### ISSUE TITLE
It has been solved.

### ISSUE THREAD

**Issue by lllllsytt** [Bosh-Kuo/GNN-LSTM-based-Fusion-Model-for-Structural-Dynamic-Responses-Prediction#4]

**Comment by Bosh-Kuo** [comment]
Hi @lllllsytt ,

Sorry for the late reply. I'm glad to see that you've solved the problem!

In your initial comment, you mentioned that the model's performance did not meet expectations because the validation loss remained consistently high.

Here are my thoughts:
As shown in Figure 8 of my paper, all training experiments indicate that the validation loss does not drop as low as the training loss. However, this does not necessarily mean that the model's predictive ability is poor. If you plot the predicted response history, you'll see that in most cases, the predictions are quite well. For this task, unless we have a large amount of data, it is difficult to make the validation loss approach the training loss in the convergence stage.

Hope this helps! Let me know if you have any other questions.
